src/renderer/components/SetupTab/index.jsx

```diff
@@ -401,7 +402,8 @@ export default class SetupTab extends React.Component {
                 onClick={this.browseForDatastack}
                 variant="link"
               >
-                Load parameters from file
+                <MdFolderOpen />
+                {' Load parameters from file'}

```

Comment:
I think this is supposed to be replaced by a class too?

Reply:
Whoops, thank you!